### PR TITLE
add AABBFromOrientedBox

### PR DESCRIPTION
**Pull request by eisoku9618**
Add an utility function to convert an OBB to an AABB in the reference frame of the OBB

### CODE REVIEW

File: include/openrave/geometry.h [hunk 1]
```diff
@@ -1932,6 +1932,24 @@ inline RaveOrientedBox<T> OrientedBoxFromAABB(const RaveAxisAlignedBox<T>& ab, c
     return OrientedBoxFromAABB(ab,RaveTransform<T>(t));
 }
 
+/// \brief Transform an axis aligned bounding box to an oriented bounding box expressed in transform.
+///
+/// \ingroup geometric_primitives
+/// \param[in] t transformation used to set the coordinate system of ab.
+template <typename T>
+inline RaveAxisAlignedBox<T> AABBFromOrientedBox(RaveOrientedBox<T> obb)
```
Comment (yoshikikanemoto) [inline include/openrave/geometry.h:1940]: @eisoku9618 use const reference

File: include/openrave/geometry.h [hunk 2]
```diff
@@ -1932,6 +1932,24 @@ inline RaveOrientedBox<T> OrientedBoxFromAABB(const RaveAxisAlignedBox<T>& ab, c
     return OrientedBoxFromAABB(ab,RaveTransform<T>(t));
 }
 
+/// \brief Transform an axis aligned bounding box to an oriented bounding box expressed in transform.
+///
+/// \ingroup geometric_primitives
+/// \param[in] t transformation used to set the coordinate system of ab.
```
Comment (yoshikikanemoto) [inline include/openrave/geometry.h:1938]: fix doc

**Comment by cielavenir** [comment]
found a typo, fixed in https://github.com/rdiankov/openrave/pull/1331

**Comment by eisoku9618** [comment]
@cielavenir @jschaare Just clarification, this typo is not from the main part of this PR but from https://github.com/rdiankov/openrave/pull/1329

**Pull request merged**